bin/server.js

```diff
@@ -56,7 +55,6 @@ const properties = new Properties();
 const sources = new Sources(properties);
 
 // Add metadata and some statics
-properties.dynamic(new Consul('consul', Config.get('consul')), 'consul');
```

Comment:
It has to be specified in the index like [this](https://github.com/rapid7/propsd/blob/781799eec8343f124dea0f7323fbb365ebac8fd5/test/data/s3/index.json#L32-L35).